project/CodeCoverageSettings.scala

```diff
@@ -16,7 +16,7 @@ object CodeCoverageSettings {
 
   val settings: Seq[Setting[_]] = Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimumStmtTotal := 100,
+    ScoverageKeys.coverageMinimumStmtTotal := 66,
```

Comment:
Once we start working on this, we need to bump this up to 90 (or is there a test we can write/update now that'll push this up for us?)

Reply:
Yes. It's just got the HelloWorldController in there at the moment so no point getting coverage up - just need it to get through the build pipeline.